css/Components/Hero.scss

```diff
@@ -32,7 +32,7 @@ $Hero-backgroundOverlay-color: rgba(8, 19, 64, 0.7);
 .Hero-background {
   position: absolute;
 
-  top: -$Hero-navbar-height;
+  top: -100px;
```

Comment:
Did you test on all resolutions?

Reply:
Yes. Worst case scenario is that this cuts off a bit of the video at the top, which is a very good compromise in my opinion